app/helpers/form_helper.rb

```diff
@@ -192,6 +192,19 @@ def autocomplete_f(f, attr, options = {})
     end
   end
 
+  def byte_size_f(f, attr, options = {})
+    base_id = ActionView::Helpers::Tags::TextField.new(f.object_name, attr, nil, options).send :add_default_name_and_id, {}
+    options[:class] = options[:class].to_s + ' byte_spinner'
+    options[:help_inline] ||= popover('', _("When specifying custom value, add 'MB' or 'GB' at the end. Field is not case sensitive and MB is default if unspecified."))
+    options[:help_block] ||= content_tag(:span, :class => 'maximum-limit hidden') do
+      content_tag(:i, '', :class => 'pficon-warning-triangle-o') +
+      content_tag(:span, ' ' + _('Specified value is higher than recommended maximum'), :class => 'error-message')
+    end
+    options[:onchange] = "$(this).byte_spinner('validate'); #{options[:onchange]}"
```

Comment:
you have `onchange` here, you also add `onchange` listener below in the initialization, would it make sense to have one `changed` function that handles everything instead of multiple ones being called?